bin/build-mock-web-services

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
+
+# DOC: Build a new docker image for running the mock web services container and optionally push to Docker Hub if PUSH_IMAGE is set to anything. Builds for "linux/amd64" unless PLATFORM is set
+
+source bin/lib.sh
+
+IMAGE="mock-web-services"
+LOCATION="mock-web-services/"
+DOCKERFILE="mock-web-services/mock-web-services.Dockerfile"
+
+BUILD_ARGS="-f ${DOCKERFILE}
+  -t civiform/${IMAGE}:latest
+  --cache-from civiform/${IMAGE}
+  --build-arg BUILDKIT_INLINE_CACHE=1
+  --build-context server=server/
+  ${LOCATION}"
+
+PLATFORM_ARG=()
+if [[ "${PLATFORM}" ]]; then
+  PLATFORM_ARG=(--platform "${PLATFORM}")
+fi
+
+# Build the multi-platform image
+echo "start ${IMAGE} build"
+cmd="docker buildx build ${PLATFORM_ARG[@]} ${BUILD_ARGS[@]}"
+$cmd
```

Comment:
**question (blocking):** why assign the command to a variable first instead of `docker buildx build "${PLATFORM_ARG[@]}" "${BUILD_ARGS[@]}"`?

Reply:
I'd have to look. All the existing ones are this way and I just copied it over. I'd rather they maintain the same format so if I start changing one I'd rather clean them all up in a separate PR.